.github/workflows/purge-js-delivr-cache.yml

```diff
@@ -0,0 +1,25 @@
+# Temporary trigger workflow on PR to test the job
+#on:
+#  workflow_dispatch:
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test-curl-action:
+    name: "Curl to purge JSDelivr cache"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Curl"
+        uses: indiesdev/curl@v1.1
+        with:
+          url: https://purge.jsdelivr.net/
+          method: "POST"
+          accept: 200,201,204,202
+          headers: '{ "cache-control": "no-cache", "content-type": "application/json" }'
+          body: "{'path': ['/npm/@alma/widgets@3.x.x/dist/widgets.umd.js','/npm/@alma/widgets@3.x/dist/widgets.umd.js','/npm/@alma/widgets@3.x.x/dist/widgets.module.css','/npm/@alma/widgets@3.x/dist/widgets.module.css','/npm/@alma/widgets@3.x.x/dist/widgets.css','/npm/@alma/widgets@3.x/dist/widgets.css','/npm/@alma/widgets@3.x.x/dist/widgets.min.css','/npm/@alma/widgets@3.x/dist/widgets.min.css']}"
```

Comment:
Do you think, there is a way to update the 3 with the current version ? I dunno how to do so easily

Reply:
I'm not sure how to get the current version from the CI, that can be an improvement for another PR, that is a a good idea